index.html

```diff
@@ -30,6 +33,175 @@ <h2>I’am a Software Developer</h2>
             </div>
         </div>
     </section>
+    <!-- the hero section ends here -->
+
+    <!-- the works section -->
+    <section>
+        <div class="works-section">
+            <div class="recentwork">
+                <p>My Recent Works</p>
+            </div>
+
+            <div class="workone">
+                <img src="./images/workone.png" alt="">
+                <h2>Multi-Post Stories</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <div class="stack">
+                    <h3>css</h3>
+                    <h3>html</h3>
+                    <h3>bootstrap</h3>
+                    <h3>Ruby</h3>
+                </div>
+                <button>See Project</button>
+            </div>
```

Comment:
- [ ] Dear @chibunduonyeje1, Please use a `a` or `button` tag for your `See Project` instead of a `div` tag. Remember in the future you are going to use this to link to the project also this will provide a semantic-wise advantage.